src/rgw/setup/radosgw_start

```diff
@@ -3,6 +3,10 @@
 INDEX=$1
 CONFIG_FILE=$2
 LOG_FILE=$3
+rgw_crash_dir=$4
+
+echo "Enable rgw core file generation at $rgw_crash_dir"
```

Comment:
echo "Change RGW CWD to dir $rgw_crash_dir, to retain core files in there"

Reply:
done